DotNetRu.Clients.Portable/ViewModel/SpeakersViewModel.cs

```diff
@@ -70,7 +74,44 @@ orderby speaker.FullName
 		public ICommand LoadSpeakersCommand =>
 			loadSpeakersCommand ?? (loadSpeakersCommand = new Command(async (f) => await ExecuteLoadSpeakersAsync((bool) f)));
 
-		async Task<bool> ExecuteLoadSpeakersAsync(bool force = false)
+	    private static IEnumerable<Speaker> GetSpeakers()
```

Comment:
Return List<Speaker> - it's more efficient as the caller won't need to enumerate the collection